### PR TITLE
Additional documentation linked

### PR DESCRIPTION
**Pull request by hathin**
Documentation with current Intune network endpoints added

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/managed-desktop/get-ready/network.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ Windows 10 Enterprise including Windows Update for Business | [Manage connection
 Delivery Optimization | [Configure Delivery Optimization for Windows 10 updates](https://docs.microsoft.com/windows/deployment/update/waas-delivery-optimization)
 Office 365 | [Office 365 URL and IP address ranges](https://docs.microsoft.com/office365/enterprise/urls-and-ip-address-ranges)
 Azure Active Directory | [Hybrid identity required ports and protocols](https://docs.microsoft.com/azure/active-directory/hybrid/reference-connect-ports) and [Active Directory and Active Directory Domain Services Port Requirements](https://aka.ms/AA26ygm) 
-Microsoft Intune | [Intune network configuration requirements](https://docs.microsoft.com/intune/network-bandwidth-use)
+Microsoft Intune | [Intune network configuration requirements](https://docs.microsoft.com/intune/network-bandwidth-use)<br>[Network endpoints for Microsoft Intune](https://docs.microsoft.com/en-us/mem/intune/fundamentals/intune-endpoints)
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/get-ready/network.md]:
```suggestion
Microsoft Intune | [Intune network configuration requirements](https://docs.microsoft.com/intune/network-bandwidth-use)<br>[Network endpoints for Microsoft Intune](https://docs.microsoft.com/mem/intune/fundamentals/intune-endpoints)
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review - please remove locale from link.

**Comment by hathin** [comment]
acknowledged review

**Comment by mypil** [comment]
@hathin - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by mypil** [comment]
@jaimeo - PR has been copy edited and is ready for final review. 

Thank you.

cc: @kenwith

**Pull request merged**